code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -864,3 +864,17 @@
 		return
 	transmit = !transmit
 	to_chat(user, "<span class='notice'>You [transmit ? "enable" : "disable"] the board's automatic disposal transmission.</span>")
+
+/obj/item/circuitboard/machine/stacking_unit_console
+	name = "Stacking Machine Console (Machine Board)"
+	build_path = /obj/machinery/mineral/stacking_unit_console
+	req_components = list(
+		/obj/item/stack/sheet/glass = 2,
+		/obj/item/stock_parts/manipulator = 2)
```

Comment:
This is pretty much just a computer; the stacker does all the work. Why not just glass and cable coil?

Reply:
why didnt I think of that hold on